lib/run-uws-tracker.ts

```diff
@@ -235,6 +241,35 @@ function buildServer(
                     memory: process.memoryUsage(),
                 }));
         },
+    ).get(
+        "/*",
+        (response: HttpResponse, request: HttpRequest) => {
+            debugRequest(server, request);
+
+            const path = request.getUrl();
+
+            let data = undefined;
+
+            try {
+                data = readFileSync(`${process.cwd()}${path}`);
```

Comment:
Serving files is very responsible. This line is a potential vulnerability that allows reading arbitrary files in the file system.

Reply:
Instead of just clossing the PR, you can suggest a proper fix for this... I think resolve the absolute path and ensure falls under the current working dir would be enough.